src/ray/raylet/lineage_cache.cc

```diff
@@ -0,0 +1,39 @@
+#ifndef LINEAGE_CACHE_CC
+#define LINEAGE_CACHE_CC
+
+#include "lineage_cache.h"
+
+#include <stdexcept>
+
+namespace ray {
+
+LineageCache::LineageCache() {}
+
+ray::Status LineageCache::AddTask(const Task &task) {
+  throw std::runtime_error("method not implemented");
```

Comment:
Please do not use exception.  Use `Status::NotImplementedError` (or unimplemented).

Reply:
this is temporary and meant to be replaced. The intention is for these methods to serve as placeholders for near future implementation (shortly after the merge).